server/src/database/reactions.go

```diff
@@ -0,0 +1,88 @@
+package database
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/uptrace/bun"
+	"scrumlr.io/server/common"
+)
+
+type Reaction struct {
+	bun.BaseModel `bun:"table:reactions"`
+	ID            uuid.UUID
+	Note          uuid.UUID
+	User          uuid.UUID
+	ReactionType  string
+}
+
+type ReactionInsert struct {
+	bun.BaseModel `bun:"table:reactions"`
+	Note          uuid.UUID
+	User          uuid.UUID
+	ReactionType  string
+}
+
+type ReactionUpdate struct {
+	bun.BaseModel `bun:"table:reactions"`
+	ReactionType  string
+}
+
+// GetReaction gets a specific Reaction
+func (d *Database) GetReaction(id uuid.UUID) (Reaction, error) {
+	var reaction Reaction
+	err := d.db.
+		NewSelect().
+		Model((*Reaction)(nil)).
+		Where("id = ?", id).
+		Scan(context.Background(), &reaction)
+	return reaction, err
+}
+
+// GetReactions gets all reactions for a specific board
+// query:
+// SELECT r.* FROM reactions r JOIN notes n ON r.note = n.id WHERE n.board = ?;
+func (d *Database) GetReactions(board uuid.UUID) ([]Reaction, error) {
+	var reactions []Reaction
+	err := d.db.
+		NewSelect().
+		Model(&reactions).
+		Join("JOIN notes ON notes.id = reaction.note"). // important: 'reaction.note' instead of 'reactions.note'
+		Where("notes.board = ?", board).
```

Comment:
You could solve it that way, but I don't think it's neccesarily a better solution to have a board id entry in every reaction entity.  
I know using JOINS has a bad reputation but since the board id is indexed I don't really see a problem.